modules/data.remote/R/NLCD.R

```diff
@@ -69,8 +69,7 @@ download.NLCD <- function(outdir, year = 2011, con = NULL) {
 ##' @description Based on codes from Christy Rollinson and from Max Joseph (http://mbjoseph.github.io/2014/11/08/nlcd.html)
 extract_NLCD <- function(buffer, coords, data_dir = NULL, con = NULL, year = 2011) {
     library(raster)
-    require(rgdal)
```

Comment:
Do we know if raster's dependencies on rgdal have been fixed? If not, this fix could be pretty short lived. I know in other cases we're trying to migrate from raster to terra, but that's a harder fix than the current PR

Reply:
Ah right, I don't know much about `raster`, but checking `terra` some lines can be changed, e.g.:

```
nlcd <- raster::raster(filename)
```
to
```
nlcd <- terra::rast(filename)
```
and
```
sites <- sp::spTransform(sites, raster::crs(nlcd))
```
to
```
sites <- sp::spTransform(sites, terra::crs(nlcd, proj=TRUE))
```
then probably instead of `raster::extract`, `terra::extract` could be used, but I'm not familiar with this function. Is there someone who can quickly check if such changes would work as intended?

Reply:
@istfer Given that this function isn't actively being used in any automated workflow at the moment (i.e. there's no one else who has things set up for a quick and easy test), I'd be OK with you making the obvious changes and leaving a note or warning that they haven't been rigorously tested.  

FYI the closest analog is @JoshuaPloshay work on PR https://github.com/PecanProject/pecan/pull/3211 which uses terra to extract a number of other data products.

Reply:
got it, I made an attempt to replace with terra functions (I found an nlcd image to test it with) and put a warning

Reply:
`raster` no longer imports `rgdal`.  `raster` has replaced `rgdal` imports with `terra` so you do not need to change this (but probably good to do it eventually).  If a packages is on CRAN still, it no longer uses the retired r-spatial packages. You can also  check if you've gotten rid of all the `rgdal` dependencies with `pak::local_deps_explain("rgdal", "modules/data.land")`